js/context.js

```diff
@@ -7,19 +7,39 @@ module.exports = {
   '@context': [
     'https://www.w3.org/2018/credentials/v1',
     {
+      '@version': 1.1,
+      '@protected': true,
       id: '@id',
       type: '@type',
       refresh2021: 'https://w3id.org/vc-refresh-2021#',
       url: 'http://schema.org/url',
-      validAfter: {'@id': 'refresh2021:validAfter', '@type': 'xsd:dateTime'},
-      validUntil: {'@id': 'refresh2021:validUntil', '@type': 'xsd:dateTime'},
       MediatedRefresh2021: {
         '@id': 'refresh2021:MediatedRefresh2021',
-        '@type': '@id'
+        '@type': '@id',
+        '@context': {
+          validAfter: {
```

Comment:
Issues related to the spec rather than this context:
- VC vocab is moving to `validFrom` rather than `validAfter`
- It's questionable if this spec should be defining `validAfter/From` and `validUntil` to different values than the VC spec.  They are scoped, so it's not incorrect, but it could cause confusion.